packages/core/app/components/cms/GlobalSections.tsx

```diff
@@ -0,0 +1,47 @@
+import { Section } from '@vtex/client-cms'
```

Comment:
suggestion: I don't know if you're already doing that, but my suggestion is to create a task on Jira to add a list of those files that by the end of the migration we should delete. What do you think? During the migration this would be updated.

Reply:
Agreed!
Also, there are some files (most from the sdk folder) that are used for a lot of sections, and for the sake of keeping small changes per PR, I have decided to keep those files in the src folder yet. But we must migrate at the end as well. (I'll create this 2 tasks)